perception/simple_object_merger/README.md

```diff
@@ -0,0 +1,46 @@
+# simple_object_merger
+
+This package can merge multiple topics of [autoware_auto_perception_msgs/msg/DetectedObject](https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs/-/blob/master/autoware_auto_perception_msgs/msg/DetectedObject.idl) without data association algorithm.
+
+## Algorithm
+
+### Background
+
+This package can merge multiple DetectedObjects without matching processing.
+[Object_merger](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/object_merger) solve data association algorithm like Hungarian algorithm for matching problem, but it needs computational cost.
+In addition, for now, [object_merger](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/object_merger) can handle only 2 DetectedObjects topics and cannot handle more than 2 topics in one node.
+To merge 6 DetectedObjects topics, 6 [object_merger](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/object_merger) nodes need to stand.
+
+So this package aim to merge DetectedObjects simply.
+This package do not use data association algorithm to reduce the computational cost, and it can handle more than 2 topics in one node to prevent launching a large number of nodes.
+
+### Limitation
+
```

Comment:
I think this package has some assumptions:
- all input object has same(close) timestamp
- none of inputs will drop

It's better to write down these assumptions in README.

Reply:
> To care sensor objects drops or delayed

I think setting timeout parameter is reasonable for this package and error state should be handled by diagnostic layer.

> all input object has same(close) timestamp

Some radar run independently, so this assumption cannot be satisfied.

Reply:
I added timeout parameter and update README.md at [a9fb023](https://github.com/autowarefoundation/autoware.universe/pull/4219/commits/a9fb02376cfb04cd9e32abe70808a819acc04e57)